.github/workflows/build-and-test-centos-7.yml

```diff
@@ -2,10 +2,10 @@
 name: Build & Test (CentOS 7)
 
 on:
-  pull_request:
   push:
     branches:
```

Comment:
I see you included `develop` for `Fedora 34` and not here. Not really important, just something I noticed.

Reply:
Yes, thanks, this is on purpose. We can maintain this long run to make sure we don't break things on `master` but not run it in other places.